projects/igniteui-angular/src/lib/grids/selection/selection.service.ts

```diff
@@ -161,6 +161,7 @@ export class IgxGridCRUDService {
 
     begin(cell): void {
         const newCell = this.createCell(cell);
+        cell.focused = true;
```

Comment:
Why is this here? The `cell` in this context is not the IgxGridCellComponent. Moreover it does not have a focused property at all.